content/en/docs/collector/configuration.md

```diff
@@ -369,6 +369,8 @@ exporters:
     protocol_version: 2.0.0
 
   # Data sources: traces, metrics, logs
+  # NOTE: Prior to v0.86.0 use `logging` and `loglevel: debug`
+  #       instead of `debug` and `verbosity: detailed`
```

Comment:
`verbosity` is available since v0.63.0 (which was released a little less than a year ago), I don't think we have to mention it here.
```suggestion
  # NOTE: Prior to v0.86.0 use `logging` instead of `debug`
```